CONTRIBUTING.md

```diff
@@ -150,6 +151,85 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Profiling
+Profiling can be used to monitor the impact of Secretless on CPU and Memory consumption. Currently, Secretless supports two types- CPU and Memory.
+
+**Prerequisites:**
+- [Graphviz](https://graphviz.gitlab.io/download/) to visualize profiling results
+- [Postgresql](https://www.postgresql.org/download/) to install Postgres
+
+In order to configure individual profiles properly and receive desired results, perform the following:
+1. [Build](https://conjurhq.slack.com/archives/G7XCMCEAV/p1544522629029800) Secretless locally
```

Comment:
I don't think this should be a slack link - it should point to `#building`, right?